.github/workflows/main.yaml

```
@@ -16,6 +16,10 @@ concurrency:
   group: $-$
   cancel-in-progress: true
 
+permissions:
```

actionlint (1.7.4)

key "permissions" is duplicated in workflow. previously defined at line:10,col:1

(syntax-check)